### DISCUSSION TITLE
I Traveled the World with my Favorite Late Actor Robin Williams

### DISCUSSION THREAD

**Discussion by ItsNoted** [lllyasviel/Fooocus#1062] [Show and tell]
Fooocus has come so far. What we are witnessing with AI today is truly remarkable. I hope you enjoy this tour with Robin Williams I created using Fooocus!

https://mindrenders.com/robin-williams-thanks-to-fooocus

**Comment by lgwjames** [comment]
great images, a few are a bit off, and look like another actor , but great work !
